README.md

```diff
@@ -1 +1,63 @@
 # Tiidelab-Cohort-4
+> A personal professional portfolio of Victor Ade-Samuel
+
+
+## Built With
+
+- HTML5
+- CSS3
+-Javacsript
+
+
+## Live Demo 
+
+[Live Demo Link](https://victoradesamuel.github.io/Tiidelab-Cohort-4/)
+
+
+
+### Prerequisites
+- Good knowledge of Figma
```

Comment:
- [ ] For this project the given design is not from Figma so please change it to Behance.